Makefile

```diff
@@ -152,6 +160,40 @@ proto-gen: proto-build ## Generate all the code from the Protobuf files
 	@cp -r github.com/okp4/okp4d/x/* x/
 	@rm -rf github.com
 
+## Release:
+release-assets: release-binary-all release-checksums ## Generate release assets
+
+release-binary-all: $(RELEASE_TARGETS)
+
+$(RELEASE_TARGETS): ensure-buildx-builder
+	@GOOS=$(word 3, $(subst -, ,$@)); \
+    GOARCH=$(word 4, $(subst -, ,$@)); \
+    BINARY_NAME="okp4d-${VERSION}-$$GOOS-$$GOARCH"; \
+	echo "${COLOR_CYAN} 🎁 Building ${COLOR_GREEN}$$GOOS $$GOARCH ${COLOR_CYAN}release binary${COLOR_RESET} into ${COLOR_YELLOW}${RELEASE_FOLDER}${COLOR_RESET}"; \
+	docker buildx use ${DOCKER_BUILDX_BUILDER}; \
+	docker buildx build \
+		--platform $$GOOS/$$GOARCH \
+		-t $$BINARY_NAME \
+		--load \
+		.; \
+	mkdir -p ${RELEASE_FOLDER}; \
+	docker create -ti --name tmp-okp4d $$BINARY_NAME; \
+	docker cp tmp-okp4d:/usr/bin/okp4d ${RELEASE_FOLDER}/$$BINARY_NAME; \
+	docker rm -f tmp-okp4d; \
+	tar -zcvf ${RELEASE_FOLDER}/$$BINARY_NAME.tar.gz ${RELEASE_FOLDER}/$$BINARY_NAME;
+
+release-checksums:
+	@echo "${COLOR_CYAN} 🐾 Generating release binary checksums${COLOR_RESET} into ${COLOR_YELLOW}${RELEASE_FOLDER}${COLOR_RESET}"
+	@rm ${RELEASE_FOLDER}/sha256sum.txt; \
+	for asset in `ls ${RELEASE_FOLDER}`; do \
+		shasum -a 256 ${RELEASE_FOLDER}/$$asset >> ${RELEASE_FOLDER}/sha256sum.txt; \
```

Comment:
Question: is the `shasum` command always available (I mean for the common linux distros: debian, ubutuntu)? I think so, but if you could confirm it.

Reply:
@ccamel That's relevant, and that's why I just put another temporary commit to test it 😉